packages/schematics/angular/component/files/__name@dasherize@if-flat__/__name@dasherize__.__type@dasherize__.ts.template

```diff
@@ -1,9 +1,7 @@
 import { <% if(changeDetection !== 'Default') { %>ChangeDetectionStrategy, <% }%>Component<% if(!!viewEncapsulation) { %>, ViewEncapsulation<% }%> } from '@angular/core';
 
 @Component({<% if(!skipSelector) {%>
-  selector: '<%= selector %>',<%}%><% if(standalone) {%>
-  standalone: true,
-  imports: [],<%} else { %>
```

Comment:
nit: Should we leave the `imports` field?

Reply:
I left it as it was there and in most of the cases, users would need to import something.